apps/bridge/src/contexts/OFACContext.tsx

```diff
@@ -24,7 +24,7 @@ export async function fetchIsAllowed(address?: `0x${string}`): Promise<{ result:
     return { result: false };
   }
 
-  const response = await fetch(publicRuntimeConfig.complianceApiURL, {
+  const response = await fetch(publicRuntimeConfig.bridgeApiURL, {
```

Comment:
Just to make sure I understand this correctly, does the compliance endpoint run through this same API then?

Reply:
yup, just updating the name because it's doing more than just compliance now

Reply:
Ok, sweet